packages/lib/pod/src/podUtil.ts

```diff
@@ -349,7 +354,13 @@ export function deserializePODEntries(serializedEntries: string): PODEntries {
  * @returns the underlying value
  */
 export function podValueToRawValue(podValue: PODValue): PODRawValue {
-  return podValue.value;
+  if (podValue.type === "eddsa-pubkey") {
+    return `${EDDSA_PUBKEY_PREFIX}:${podValue.value}`;
+  } else if (podValue.type === "string") {
+    return `string:${podValue.value}`;
```

Comment:
Limit the extra prefix to when it's necessary.  I suggest you check if value begins with a type prefix and add `string:` to ensure no ambiguity.  Could be as simple as checking if the value contains a colon, or could be a regex to make it more selective.

Safest for forward-compatibility might be to assume a prefix of {PODName}: might be mistaken for a prefix for some future type, and add the `string` in that case, but otherwise don't.

Simplified JSON is advertised as potentially losing type information, but that's supposed to only be when the behavior is equivalent (e.g. int vs. cryptographic).  In this case string and eddsa-pubkey behave quite differently.